example/lfp/lfp.cpp

```diff
@@ -193,7 +191,7 @@ int main(int argc, char** argv) {
     const double sample_dt = 0.1; // [ms]
     const double dt = 0.1;        // [ms]
 
-    arb::simulation sim(R, arb::partition_load_balance(R, context), context);
+    arb::simulation sim(R);
```

Comment:
`Recipe` instead of `R`, while we're at it.

Reply:
That expression seems a bit dated, yes.